en/flight_controller/pixhawk6x-rt.md

```diff
@@ -198,9 +198,11 @@ It is pre-built and automatically installed by _QGroundControl_ when appropriate
 To [build PX4](../dev_setup/building_px4.md) for this target:
 
 ```
-make px4_fmu-v6x_default
+make nxp_fmurt1170-v1_default
 ```
-
+:::warning
+As of now PX4 SW enablement is done in px4 branch pr-imxrt1170-imxrt1062 only. Merge to PX4 main is expected soon.
+:::
```

Comment:
Perhaps move this warning to the top of the page?

Reply:
Good idea. Will do on Monday.

Reply:
Ping

Reply:
Agree, this should be at top of page. Anyhow the branch has changed again due to the target name change from nxp_fmu.. to px4_fmu-v6xrt. Please skip this.

Reply:
correction is done in https://github.com/NXPHoverGames/PX4-user_guide/pull/1